sci-visualization/mantid/mantid-3.3.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+FORTRAN_STANDARD=90
+PYTHON_COMPAT=python2_7
+inherit eutils cmake-utils versionator python-single-r1
+
+MAJOR_PV=$(get_version_component_range 1-2)
+
+DESCRIPTION="Utilities for processing and plotting neutron scattering data"
+HOMEPAGE="http://www.mantidproject.org/"
+SRC_URI="mirror://sourceforge/project/${PN}/${MAJOR_PV}/${P}-Source.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc opencl paraview shared-libs tcmalloc test"
+RESTRICT="test" # Testing requires sample data and X11 access
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=sci-libs/nexus-4.2[${PYTHON_USEDEP}]
+	>=dev-libs/poco-1.4.2
+	dev-libs/boost[python,${PYTHON_USEDEP}]
+	opencl?		( virtual/opencl )
+	tcmalloc?	( dev-util/google-perftools )
+	paraview?	( >=sci-visualization/paraview-4[python,${PYTHON_USEDEP}] )
+	virtual/opengl
+	dev-qt/qthelp
+	x11-libs/qscintilla
+	x11-libs/qwt:5
+	x11-libs/qwtplot3d
+	dev-python/PyQt4[${PYTHON_USEDEP}]
+	sci-libs/gsl
+	dev-python/ipython[qt4,${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]
+	dev-cpp/muParser
+	dev-libs/jsoncpp
+	dev-libs/openssl
+	sci-libs/opencascade[qt4]
+"
+
+DEPEND="${RDEPEND}
+	dev-python/sphinx
+	doc?	( app-doc/doxygen
```

Comment:
Seeing below that `doc` get mapped on `DOT`, I guess, you want to depend on `app-doc/doxygen[dot]`.